manifests/vrrp/sync_group.pp

```diff
@@ -36,6 +38,7 @@
   $group,
   Optional[Stdlib::Absolutepath] $notify_script_master_rx_lower_pri = undef,
   Array[String] $track_script                                       = [],
+  Array[String] $track_process                                      = [],
```

Comment:
are relative paths fine or should we update it to:
```suggestion
  Array[Stdlib::Absolutepath] $track_process                        = [],
```

Reply:
It's actually not a path here, but just a reference to the name of an actual track_process definition in the keepalived config.